11_hexcode/app.js

```diff
@@ -0,0 +1,20 @@
+const btn = document.getElementById("color-btn")
+const values = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, "a", "b", "c", "d", "e", "f"]
+var hex = []
+let el = 2;
+
+function hexChange() {
+    if (hex.length <= 5) {
+        for (var i = 0; i < 6; i++) {
+            el = values[Math.floor(Math.random() * values.length)];
+            hex.push(el)
+        }
+        console.log(hex)
+        document.body.style.backgroundColor = "#" + hex[0] + hex[1] + hex[2] + hex[3] + hex[4] + hex[5];
+        document.querySelector(".btn-name").innerHTML = "HEX COLOR: #" + hex[0] + hex[1] + hex[2] + hex[3] + hex[4] + hex[5]
```

Comment:
Have you learned [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)? It's easier to create dynamic strings. Give it a try!

After you tried it, there's another way more dynamic to create the hex color using only array methods`.flat()` and `.join()`

```js
const hexColor = `#${hex.flat().join('')}`
```